### PR TITLE
Pubby Maint detailing take two

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: El Tacolizard
add: Improved pubby's maint detailing
/:cl:

Basically i heard people saying pubby was bland and boring so i went through some boring looking areas and added decals like spiderwebs and grimy floors, dried vomit etc. almost all of the changes were made to sections of maint.

Example:

**Before**
<a target='_BLANK' href='http://i.imgur.com/YoxntXJ.png'></a>

**After**
<a target='_BLANK' href='http://i.imgur.com/oWU2iDP.png'></a>

### CODE REVIEW

**Comment by optimumtact** [comment]
Remove the permission change 100644 → 100755 

How did you even manage to do that?

**Comment by optimumtact** [comment]
You also need to use the map merger and ensure the file is in the tgm format.

https://tgstation13.org/wiki/Map_Merger

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 1]
```diff
@@ -1064,7 +1064,7 @@
...
 "auy" = (/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden{dir = 10},/obj/structure/cable{d1 = 2; d2 = 4; icon_state = "2-4"},/turf/open/floor/plasteel/black,/area/ai_monitored/nuke_storage)
 "auz" = (/obj/machinery/camera/motion{c_tag = "Vault"; dir = 1; network = list("SS13")},/obj/machinery/light,/obj/structure/cable{icon_state = "1-8"},/turf/open/floor/plasteel/black,/area/ai_monitored/nuke_storage)
-"auA" = (/obj/structure/safe,/obj/item/weapon/bikehorn/golden,/obj/item/ammo_box/a357,/obj/item/weapon/tank/internals/plasma/full,/obj/item/weapon/disk/fakenucleardisk,/obj/item/weapon/gun/energy/disabler,/turf/open/floor/plasteel/vault{dir = 4},/area/ai_monitored/nuke_storage)
+"auA" = (/obj/structure/safe,/obj/item/weapon/bikehorn/golden,/obj/item/ammo_box/a357,/obj/item/weapon/tank/internals/plasma/full,/obj/item/weapon/disk/fakenucleardisk,/obj/item/weapon/gun/energy/disabler,/obj/item/weapon/gun/projectile/automatic/pistol/m1911{desc = "A classic .45 handgun with a small magazine capacity. An engraving on the side reads: Tacolizard Forever"},/turf/open/floor/plasteel/vault{dir = 4},/area/ai_monitored/nuke_storage)
```
Comment (optimumtact) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: direct references to yourself are frowned upon

**Review by optimumtact — CHANGES_REQUESTED** [review]
You need to remove the direct reference to yourself

**Comment by ExcessiveUseOfCobblestone** [comment]
remove tacolizard instead of the reference

File: _maps/map_files/PubbyStation/PubbyStation.dmm (hunk 1 above)
Comment (PKPenguin321) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: why must you add pistols to the map

**Comment by pubby** [comment]
I've merged most of your changes into my local branch. Thanks. :+1:

This PR can be closed, and I'll make a new PR with these changes and several others in a few days. I'll include you in the credits.

File: _maps/map_files/PubbyStation/PubbyStation.dmm (hunk 1 above)
Comment (pubby) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: >direct references to yourself are frowned upon

`>`PubbyStation
`>`PubbyStation
`>`PubbyStation

woops

**Comment by ExcessiveUseOfCobblestone** [comment]
Mapnames and ingame items are totally different mister. 

Metastation is a reference to Metacide, the person who made the map.

**Pull request closed without merging**